services/checks/funcs_test.go

```diff
@@ -96,3 +96,132 @@ Traceback (most recent call last):
 	}}
 	assert.Equal(t, expected, res)
 }
+
+func TestAdditionalContext(t *testing.T) {
+	funcs, err := GetFuncsForVersion(1)
+	require.NoError(t, err)
+	contextFuncs := GetAdditionalContext()
+
+	testCases := []struct {
+		name   string
+		script string
+		err    string
+		result []check.Result
+	}{
+		{
+			name: "too many args",
+			script: strings.TrimSpace(`
+def check_context(rows, context):
+    func = context.get("ip_is_private", fail)
```

Comment:
Let's use the same form as we will use everywhere:
```python
ip_is_private = context.get("ip_is_private", fail)
```

That should even work for predeclared functions